src/Microsoft.DocAsCode.Build.Common/ModelAttributeHandlers/Handlers/BaseModelAttributeHandler.cs

```diff
@@ -73,6 +73,11 @@ public virtual void Handle(object obj, HandleModelAttributesContext context)
             }
         }
 
+        protected virtual bool ShouldHandle(PropInfo currentPropInfo, object declaringObject, HandleModelAttributesContext context)
+        {
+            return currentPropInfo.Attr != null;
```

Comment:
it is a protected method, and we know currentPropInfo can never be null

---
In reply to: [93932946](https://github.com/dotnet/docfx/pull/1064#discussion_r93932946) [](ancestors = 93932946)